services/QuillLMS/app/models/provider_classroom.rb

```diff
@@ -4,18 +4,21 @@
 #
 # Table name: provider_classrooms
 #
-#  id                 :bigint           not null, primary key
-#  type               :string           not null
-#  created_at         :datetime         not null
-#  updated_at         :datetime         not null
-#  canvas_instance_id :bigint
-#  classroom_id       :bigint           not null
-#  external_id        :string           not null
+#  id                     :bigint           not null, primary key
+#  provider_instance_type :string
+#  type                   :string           not null
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#  canvas_instance_id     :bigint
+#  classroom_id           :bigint           not null
+#  external_id            :string           not null
+#  provider_instance_id   :bigint
```

Comment:
Adding a polymorphic reference here, in preparation for possible future integrations